[DOC] add contributing guidelines to add figures in the specs (#679)
bids-standard/bids-specification: CONTRIBUTING.md

git commit -m 'STY: Fixed Markdown style'
```

## Adding a figure to the specifications

> A figure is worth a 1000 words! 

If you think that a figure or a picture can help summarize several aspects or notions of the 
specification, do not hesitate to make a suggestion by showing a draft in a GitHub issue.

After discussion and approval by the community, you can then submit your image
in a pull request.

Images should be added to an `images` folder that is at the same level as the Markdown file
where your image will be added. For example if you want to add a figure `figure01.png` to 
`src/05-derivatives/01-introduction.md` then your image should go to
`src/05-derivatives/images/figure01.png`.

Figures can be inserted in a Markdown like this (see also
[Markdown-Cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#images)):

```markdown
![text to show if image does not load](relative_path_to_file "text to show when hovering over image")
```
### Recommendations for figures

> 1. Try to keep the file size of your figure relatively small (inferior to 500 Ko) 

Reviewer: did you mean this?

```suggestion
1. Try to keep the file size of your figure relatively small (smaller than 500 Kb) 
```

to keep the repository light and reduce the load time of the specs 
for people who do not necessarily have broad-band internet.

> 1. Figures in the main part of the specification should aim to be for very "comprehensive" 
> but "smaller" figures can find their home in the appendices or the BIDS-starter-kit.

Reviewer: ```suggestion
1. Figures in the main part of the specification should aim to be very "comprehensive" 
but "smaller" figures can find their home in the appendices or the BIDS-starter-kit.
```

> 1. If you are adding a figure (and not picture) make sure to also join a vector format 
> of that figure (ideally as an `.svg` file) as this makes it easier to edit it in the
> future.

Reviewer: ```suggestion
1. If you are adding a figure (and not picture) make sure to also supply a vector format 
of that figure (ideally as an `.svg` file) as this makes it easier to edit it in the
future.
```
